keps/0004-add-testing-infrastructure.md

```diff
@@ -93,11 +147,16 @@ All Pull Requests into master need to have the following checks pass. These shou
 1. `go lint` and `go vet` both pass
```

Comment:
what is actually run here is `make check-formatting` which runs `go vet`, `go lint` and  staticcheck
I know this isn't part of your PR but could we correct this while we are here?

Reply:
Done!